lib/src/reddit.dart

```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2017, the Dart Reddit API Wrapper project authors.
+// Copyright (c) 2019, the Dart Reddit API Wrapper project authors.
```

Comment:
The copyright years for files in this repository reflect the year the file was created. No need to update it to 2019 here.

Reply:
Ahh, I understand it now. Mixed it up with creating a new file :)